<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="live_tab">
     <attribute name="title">
      <string>Live Feed</string>
     </attribute>
     <widget class="QWidget" name="live_widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>601</width>
        <height>351</height>
       </rect>
      </property>
      <widget class="QLabel" name="displayText">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>No USB Camera Input Detected</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="recordButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="filepathText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>601</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Saving video to: </string>
      </property>
     </widget>
     <widget class="QGroupBox" name="triggerGroup">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>130</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Record Upon:</string>
      </property>
      <widget class="QCheckBox" name="motionCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Motion</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="subjectCheck">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Person, Pets</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
